Fixes sample scripts for v8

milanholemans opened the pull request:
Aligns sample scripts with v8

Adam-it reviewed (changes requested):
@milanholemans first think I noticed is that we correct the sample to align to the latest version of CLI but we do not align the version number 🤔.
Is this done on purpose? 
IMO it is a mistake. When we change the sample code to be aligned with v8 of CLI for M365 we should also in the `assets\sample.json` update the `metadata` prop and there we specify the CLI version. I'll explain why:
when someone will look up this script sample in the [MS adoption sample gallery](https://adoption.microsoft.com/en-us/sample-solution-gallery) then he or she will see the CLI version there. They may assume that this script will run with this exact version of CLI but after our change actually it will not and it will run with latest
![image](https://github.com/user-attachments/assets/0ab376cf-549d-4092-9971-8f432b6a7047)

milanholemans commented:
Yes good catch @Adam-it, forgot to increase the number indeed.

milanholemans commented:
Hi @Adam-it, in fact, the stuff I changed has always been available in the result (at least for `flow list`).
For `flow list`, we removed `displayName`, but `properties/displayName` has always been available.
Same for the other stuff I fixed, it seems like the version label was correct there.

Adam-it commented:
> Hi @Adam-it, in fact, the stuff I changed has always been available in the result (at least for `flow list`). For `flow list`, we removed `displayName`, but `properties/displayName` has always been available. Same for the other stuff I fixed, it seems like the version label was correct there.

aaa tricky 😉but valid point.. ok.. I was going to do something else but this is quite important so will recheck ASAP 👍

Adam-it commented on docs/docs/sample-scripts/flow/export-flow-logicapp/index.mdx:
````diff
@@ -37,7 +37,7 @@ This script will export the Power Automate flow *Your sample test flow*, make su
   ```bash
   #!/bin/bash
-  ENV_NAME=m365 flow environment list --query '[?contains(displayName,`default`)] .name'
+  ENV_NAME=m365 flow environment get --query 'name'
````
I would also add `output` mode to `json` as script sample should not assume we use some output mode. For example someone might have `text` set as default and then this would probably break

```suggestion
  ENV_NAME=m365 flow environment get -o json --query 'name'
```

Adam-it reviewed (approved):
👍

Adam-it commented:
ready to merge 🚀 with small fixup

Adam-it commented:
merged manually 👏
